salt_lsp/workspace.py

```diff
@@ -206,9 +213,9 @@ def setup_custom_workspace(self):
         """
         if not isinstance(self.workspace, SlsFileWorkspace):
             old_ws = self.workspace
-            self.workspace = SlsFileWorkspace(
+            self._workspace = SlsFileWorkspace(
```

Comment:
This change will break all the functions that assume that `SaltLspProto.workspace` is an instance of `SlsFileWorkspace`